Packs/TOPdesk/Integrations/TOPdesk/TOPdesk.yml

```diff
@@ -2136,6 +2136,51 @@ script:
     - contextPath: TOPdesk.Incident.ExternalLinks.Date
       description: TOPdesk incident's external links date.
       type: Date
+  - arguments:
+    - description: The offset at which to start listing the operators at. Must be greater or equal to 0. Default is 0.
+      name: start
+      defaultValue: 0
+    - description: The amount of operators to be returned per request. Must be between 1 and 100. Default is 10.
+      name: page_size
+      defaultValue: 50
+    - description: Supports a set of OData filters which can be combined with 'and' and 'or' operators. Filter expressions can also be grouped by parentheses. https://developers.topdesk.com/explorer/?page=assets#/Assets/getAssets.
+      name: filter
+    - description: Search assets by name.
+      name: search_term
+    - description: List only Archived or only Active devices. Leave empty for all.
+      name: archived
+      predefined:
+        - "true"
+        - "false"
+    - name: fields
+      description: Additional fields to include in the response.
+      defaultValue: ''
+      predefined:
+      - 'archived,etag,id,text'
+    description: Get list of assets.
+    name: topdesk-assets-list
+    outputs:
+      - contextPath: TOPdesk.Asset.Archived
+        description: Asset status, archived or not
+        type: Boolean
+      - contextPath: TOPdesk.Asset.Etag
+        description: Asset entry date
+        type: String
+      - contextPath: TOPdesk.Asset.Id
+        description: Unique ID of the Asset
+        type: String
+      - contextPath: TOPdesk.Asset.Text
+        description: Asset freetext field, often the name of the asset
+        type: String
+  - arguments:
+    - description: The Unique ID of the Asset
+      name: asset_id
+      required: true
+    - description: "A json with the fields that you want to configure: {'asset_name':'testasset'}"
```

Comment:
A JSON format with the fields that you want to configure: {'asset_name':'testasset'}"

Reply:
@richardbluestone I am not sure what to do with this, sorry